tests/hydra.rs

```diff
@@ -440,3 +452,209 @@ fn scenario_2() -> TestResult {
     ];
     test_scenario(payloads, "tests/hydra/scenario_2.txt")
 }
+
+#[test]
+fn hydra_emulation_scenario_1() -> TestResult {
+    hydra_emulation_test("tests/hydra/scenario_1.txt".to_string())
+}
+
+#[test]
+fn hydra_emulation_scenario_2() -> TestResult {
+    hydra_emulation_test("tests/hydra/scenario_2.txt".to_string())
```

Comment:
I don't understand, what is the purpose of the two `hydra_emulation_test` tests which have a clustom websocket client as opposed to testing `oura`? If this is a sanity-check from development I don't think it is needed

Reply:
removed those redundant tests - although were of use to me when I built the final one as an intermediate steps